tests/functional/aws-node-sdk/test/bucket/getLocation.js

```diff
@@ -70,27 +70,12 @@ describeSkipAWS('GET bucket location ', () => {
             });
         });
 
-        describe('without location configuration', () => {
```

Comment:
Removed this test for redundancy because it's done more dynamically in multipleBackend/put.js